app/views/layouts/_nav.html.erb

```diff
@@ -16,34 +14,37 @@
         <h3 class="home_app_title our-custom-header"><a href="#" data-activates="slide-out" class="menu-collapse"><i class="mdi-navigation-menu" style="position: absolute; top: 0px"></i></a></h3>
       </li>
     </ul>
+
     <ul class="right">
       <% if content_for? :show %>
         <%= yield :show %>
       <% end %>
 
       <% if current_user %>
-      <li>
-        <a class="<%= "dropdown-button" unless @current_page && @current_page == "new"%>" data-activates="dropdown-user_option" data-beloworigin="true" href="#" style="padding:0 20px;">
-          <% if current_user.profile_url.present? %>
-          <%= image_tag current_user.profile_url, class: "profile_pic circle responsive-img" %>
-          <% else %>
-          <%= image_tag "defaultimage.jpeg", class: "left profile_pic circle responsive-img" %>
-          <% end %>
-          <%= current_user.first_name.split(" ")[0] %>
-          <i class=" our-fa-caret-down fa fa-caret-down" style="margin-left: 4px; font-size: 15px; float: right;"></i>
-        </a>
-      </li>
-      <ul class="dropdown-content right" id="dropdown-user_option">
-        <li><%= link_to "My Account", dashboard_path, data: {no_turbolink: true}, class: "our-dropdown-content" %></li>
-        <li><%= link_to "Logout", signout_path, data: {no_turbolink: true}, class: "our-dropdown-content" %></li>
-      </ul>
+        <li>
+          <a class="<%= "dropdown-button" unless @current_page && @current_page == "new"%>" data-activates="dropdown-user_option" data-beloworigin="true" href="#" style="padding:0 20px;">
+            <% if current_user.profile_url.present? %>
+            <%= image_tag current_user.profile_url, class: "profile_pic circle responsive-img" %>
+            <% else %>
+            <%= image_tag "defaultimage.jpeg", class: "left profile_pic circle responsive-img" %>
+            <% end %>
+            <%= current_user.first_name.split(" ")[0] %>
```

Comment:
I don't understand why you are splitting the first name. I don't know anybody with a name that has space in it.